src/diffusers/models/unet_2d_condition.py

```diff
@@ -371,6 +393,9 @@ def forward(
             class_emb = self.class_embedding(class_labels).to(dtype=self.dtype)
             emb = emb + class_emb
 
+        if additive_time_embeddings is not None:
```

Comment:
actually let's put this instead into the class embeddings. Will help you here

Reply:
Is that feasible? The class embeddings are an embeddings layer and we compute the additive time embeddings by projecting the text and image embeddings here https://github.com/huggingface/diffusers/blob/a866ded8a1523b9270ebac8972a1709b94d2c069/src/diffusers/pipelines/unclip/text_proj.py#L52-L56